### ISSUE TITLE
I got stopped by unknown force at middle or possible cheater, please check replay

### ISSUE THREAD

**Issue by lousyfear** [ValveSoftware/Dota2-Gameplay#5520]
### Description

Please check the replay I got stopped or blocked by unknown force or he is cheating I put the timestamp below and screenshot. I was trying to get uphill and blocked by the unknown. If you can see I click on high ground and stopped and forced to fight him instead.

### Example Match ID (and possibly Timestamp)

6872160341 - 02:36

### Screenshots

![Screenshot 2022-11-18 180729](https://user-images.githubusercontent.com/113813905/202829348-eb9d40e9-f3d8-4c32-b554-a7f2de79052e.jpg)

**Comment by lousyfear** [comment]
Because of this I lost the match cause I am tilted but I understand sometimes there is a bugged. I hope it get fixed soon

**Comment by tomsawyerr** [comment]
YES, THEY ENDORSE CHEATERS, GRIEFERS, AS LONG AS THEY PURCHASE SHIT FROM THE STORE.
